### ISSUE TITLE
Custom styling issues

### ISSUE THREAD

**Issue by eqlion** [n4kz/react-native-material-dropdown#223]
As far as I understand, at the moment it is impossible to change the color of the label. What is more, the theming is rather difficult in general, perhaps overhaul is due.

**Comment by muazniwas** [comment]
Agreed. And also it would be much better if we could add more style to the text field. textColor prop is allowed which can only be used to change the color of the text. Having other styling options like fontWeight or fontSize would be awesome.
Thank you
